File: java-spring/transactions-command-side-service/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/main/TransactionsCommandSideServiceMain.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.main;

import net.chrisrichardson.eventstore.javaexamples.banking.web.TransactionsCommandSideServiceConfiguration;
import org.springframework.boot.SpringApplication;

public class TransactionsCommandSideServiceMain {

  public static void main(String[] args) {
    SpringApplication.run(TransactionsCommandSideServiceConfiguration.class, args);
  }
}

File: scala-spring/transactions-command-side-service/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/main/TransactionsCommandSideServiceMain.scala
package net.chrisrichardson.eventstore.examples.bank.web.main

import net.chrisrichardson.eventstore.examples.bank.web.TransactionsCommandSideServiceConfiguration
import org.springframework.boot.SpringApplication

object TransactionsCommandSideServiceMain  {

  def main(args: Array[String]) : Unit = SpringApplication.run(classOf[TransactionsCommandSideServiceConfiguration], args :_ *)

}
